app/lib/actions/katello/product/create.rb

```diff
@@ -34,6 +34,7 @@ def finalize
           product = ::Katello::Product.find(input[:product][:id])
           product.cp_id = input[:cp_id]
           product.save!
+          product.import_subscription(input[:subscription_id])
```

Comment:
it does but what I believe is happening is the appropriate association
isn't being created as the product does not have a candlepin id yet.

John Mitsch
Red Hat Engineering
(860)-967-7285
irc: jomitsch

On Thu, Aug 4, 2016 at 9:53 AM, Justin Sherrill <notifications@github.com>
wrote:

> In app/lib/actions/katello/product/create.rb
> <https://github.com/Katello/katello/pull/6227#discussion_r73525437>:
>
> > @@ -34,6 +34,7 @@ def finalize
> >            product = ::Katello::Product.find(input[:product][:id])
> >            product.cp_id = input[:cp_id]
> >            product.save!
> > +          product.import_subscription(input[:subscription_id])
>
> Doesn't this happen in the ReindexSubscriptions call above?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Katello/katello/pull/6227/files/f039347342931d6c4658846e1eef36fe7760af07#r73525437>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFsCrvx2IK6rFioSk7lnLZQi1j13AjOOks5qce7KgaJpZM4JctAJ>
> .
>